src/App.jsx

```diff
@@ -138,15 +139,30 @@ function App() {
     };
 
     return (
-        <>
-            <h1>Todo List</h1>
-            <AddTodoForm onAddTodo={addTodo} />
-            {isLoading ? (
-                <p>Loading...</p>
-            ) : (
-                <TodoList todoList={todoList} onRemoveTodo={removeTodo} />
-            )}
-        </>
+        <BrowserRouter>
```

Comment:
I agree that the instructions were lacking in this assignment! To answer your questions about where to put content, anything that is inside your route will only show when you are on that specific page, and anything that you put outside of our routes will be shown on all pages. I don't think it's good or bad necessarily to put html tags outside your routes like you've done here, just depends on what you want.

Once use case for putting content outside the routes is if you have a navigation bar at the top or on the side of your screen. You probably want those on all your pages, not just one. In that case, putting it outside your routes makes sense. Here you put your `<h1>` outside. If you think that's something that should be on multiple pages, go for it (but obviously this assignment is tiny and the second page is not important, so there's no strong reasoning here to go either way).

Reply:
I tried to put <h1> just below the Route and it got error so I moved it out below the Routes and it didn't have an error. But I didn't this to appear on all pages. I wasn't sure which layer(Routes, route) to put the <h1>. I tried to put in each layer and it got errors until I finally put it after <> and that's when the error is gone and this is place where I really want to put it.